test/e2e/infinispan/smoke_test.go

```diff
@@ -89,6 +103,22 @@ func verifyDefaultAuthention(require *require.Assertions, ispn *v1.Infinispan) {
 	testAuthentication(ispn, schema, user, pass)
 }
 
+// Make sure no PVCs were created
+func verifyScheduling(assert *assert.Assertions, require *require.Assertions, ispn *v1.Infinispan) {
+	pod := corev1.Pod{}
+	require.NoError(testKube.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Name: ispn.Name + "-0", Namespace: tutils.Namespace}, &pod))
+	// From Infinispan CR to pods
+	areEqual := false
+	for _, v := range pod.Spec.Tolerations {
+		if reflect.DeepEqual(v, ispn.Spec.Scheduling.Tolerations[0]) {
```

Comment:
We can just do `assert.EqualValues()` here on each of the tolerations.

Reply:
k8s adds some tolerations, so arrays are not equal. so we need to check if our toleration is in the array